src/transformers/models/longformer/modeling_tf_longformer.py

```diff
@@ -613,7 +613,6 @@ def _linear(self, inputs):
         return tf.reshape(logits, [batch_size, length, self.vocab_size])
 
 
-# Copied from transformers.models.bert.modeling_tf_bert.TFBertIntermediate
```

Comment:
Why is Longformer not included in the change?

Reply:
Because I don't know well enough this model, I prefer to let @patrickvonplaten handle this one.

Reply:
The intermediate layers of Longformer are 1-to-1 the same than BERT and there should be no problem to keep those lines. I'd be surprised if leaving this line would throw an error tbh. Did you try just leaving them? The only difference in Longformer is the Self-attention layer and all none of those copy statements concern the self-attention layer, so IMO we should leave the statements and run `make fix-copies`

Reply:
Oh indeed it works for the Intermediate layer! Only the Self attention still needs to be updated accordingly :)

Reply:
Yes that I'm happy to do in another PR - would be amazing if you could open a one-liner issue about it and tag me :-)